channels/pkg/channels/addon.go

```diff
@@ -152,3 +164,36 @@ func (a *Addon) EnsureUpdated(ctx context.Context, k8sClient kubernetes.Interfac
 
 	return required, nil
 }
+
+func (a *Addon) AddNeedsUpdateLabel(ctx context.Context, k8sClient kubernetes.Interface) error {
+	klog.Infof("addon %v wants to roll %v nodes", a.Name, a.Spec.NeedUpdate)
+	selector := ""
+	switch a.Spec.NeedUpdate {
+	case "masters":
```

Comment:
Should probably use "control-plane" for newly added fields.

Reply:
Hrm, I wonder if we should introduce a mix of old and new terminology now or instead wait for a new apiVersion to change all the terminology used in the API at once.

Reply:
I'm willing to let this proceed with the legacy terminology.

Reply:
This isn't really exposed to users, so I am all for changing the names.